[qunat][fx] Remove "additional_qat_mapping" key from prepare_custom_config_dict

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75521
* #75520
* #75469
* #75389
* #75388
* __->__ #75387
* #75386
* #75378

Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35451955](https://our.internmc.facebook.com/intern/diff/D35451955)

jerryzh168 committed:
[qunat][fx][bc-breaking] Remove "additional_qat_mapping" key from prepare_custom_config_dict

Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35451955).

jerryzh168 committed:
Update on "[qunat][fx][bc-breaking] Remove "additional_qat_mapping" key from prepare_custom_config_dict"


Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35451955](https://our.internmc.facebook.com/intern/diff/D35451955)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[qunat][fx][bc-breaking] Remove "additional_qat_mapping" key from prepare_custom_config_dict"


Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35451955](https://our.internmc.facebook.com/intern/diff/D35451955)

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35451955).

vkuzo reviewed (approved):
maybe we can update the PR title too?

jerryzh168 committed:
Update on "[qunat][fx] Remove "additional_qat_mapping" key from prepare_custom_config_dict"


Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35451955](https://our.internmc.facebook.com/intern/diff/D35451955)

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35451955).

jerryzh168 committed:
Update on "[qunat][fx] Remove "additional_qat_mapping" key from prepare_custom_config_dict"


Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35451955](https://our.internmc.facebook.com/intern/diff/D35451955)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[qunat][fx] Remove "additional_qat_mapping" key from prepare_custom_config_dict"


Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35451955](https://our.internmc.facebook.com/intern/diff/D35451955)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[qunat][fx] Remove "additional_qat_mapping" key from prepare_custom_config_dict"


Summary:
This is now replaced with backend_config_dict, we don't want to expose the implementation detail to
users. We'll have docs for backend_config_dict later

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35451955](https://our.internmc.facebook.com/intern/diff/D35451955)

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35451955).
